site/content/en/docs/concepts/preemption.md

```diff
@@ -54,11 +54,14 @@ Kueue offers two preemption algorithms. The main difference between them is the
 preemptions from a ClusterQueue to others in the Cohort, when the usage of the preempting ClusterQueue is
 already above the nominal quota. The algorithms are:
 
-- **[Classic Preemption](#classic-preemption)**: Preemption in the cohort only happens when the usage of the preempting ClusterQueue
-  will be under the nominal quota after the ongoing admission process, or when all the candidates for preemption belong to
-  the same ClusterQueue as the preempting Workload. In other words, ClusterQueues
-  can only borrow quota from others in the cohort if they do not preempt admitted Workloads from
-  other ClusterQueues. ClusterQueues in a cohort borrow resources in a first-come first-served fashion.
+- **[Classic Preemption](#classic-preemption)**: Preemption in the cohort only happens when:
+  - all the candidates for preemption belong to the same ClusterQueue as the preempting Workload 
+    or other borrowing ClusterQueues within the same cohort matching the preemptor's queue `borrowWithinCohort` policy.
+  - the usage of the preempting ClusterQueue will be under the nominal quota after the ongoing admission process
```

Comment:
nit: change order (non-borrowing first) to match `Classical Preemption` section below (or only go into detail in one place - let's see what we conclude in the comment thread above)

Reply:
done